src/sparse/KokkosSparse_spgemm_numeric.hpp

```diff
@@ -139,6 +139,8 @@ void spgemm_numeric(KernelHandle *handle,
         "If you need this case please let kokkos-kernels developers know.\n");
   }
 
+  if (m < 1 || n < 1 || k < 1) return;
```

Comment:
I do not recall the exact interface we provide so I just want to make sure that the short cut above does not create issues for something like: `C=beta*C+alpha*A*B` which would fail to compute `C=beta*C` if `k==0`.
I believe that we only implement `C=alpha*A*B` though?

Reply:
Correct, we don't support adding to an existing C, so this check is OK.